test/unit/utils_test.js

```diff
@@ -485,7 +498,7 @@ describe('Utils', function() {
         .addOperation(
           StellarSdk.Operation.manageData({
             source: clientKP.publicKey(),
-            name: "example.org auth",
+            name: "testanchor.stellar.org auth",
```

Comment:
Is this the only change? And at line 511? If so, it's fine by me. Just want to make sure I'm not missing anything.

Reply:
:+1: It's one of the two changes.